tap-client/src/components/home/HomeComponent.tsx

```diff
@@ -1,8 +1,107 @@
+import collegeImg from "../../../src/assets/home-assets/stock-TU.jpg"
+import capLottie from "../../../src/assets/home-assets/capLottie.json"
+import aiLottie from "../../../src/assets/home-assets/aiLottie.json"
+import studentsLottie from "../../../src/assets/home-assets/studentsLottie.json"
+import Lottie from "lottie-react";
+import { useNavigate } from 'react-router-dom';
+
 export default function HomeComponent() {
+
+    const navigate = useNavigate();
+
+    const handleClick = () => {
+        navigate('/about');
+    }
+
     return (
         <div>
-            <h1>Home</h1>
-            <p>Welcome to the TAP!</p>
+            <div className="flex mt-10 h-screen items-center justify-center">
+                <img src={collegeImg} alt="Lanscape photo of Towson University" className="w-[50%]"/>
+                <div className="p-5 mt-5 w-[30%] m-[1.5%] float-left">
+                    <p className="text-2xl text-center text-towsonGraphiteDark">
+                        The Mission
+                    </p>
+                    <div className="w-full p-8 text-center h-[200px] text-towsonGraphiteDark">
+                        Towson Academic Pathway is a student-made and student-run service. We are dedicated to 
+                        providing exceptional planning services to help you achieve your goals during your
+                        time at Towson University. 
+
+                        <br />
+
+                        <button
+                            onClick={handleClick}
+                            className="px-3 py-1 mt-3 bg-towsonGold text-towsonWhite font-medium rounded-lg hover:bg-towsonGoldDark transition"
+                        >
+                            Meet the Team
+                        </button>
+                    </div>
+                </div>
+            </div>
+
+            <div className="flex mt-10 h-screen items-center justify-center">
+                <div className="p-3 mt-5 w-[30%] float-left">
+                    <p className="text-3xl text-center text-towsonGraphiteDark">
+                        Plan your Degree
+                    </p>
+                    <div className="w-full p-8 text-center h-[200px] text-towsonGraphiteDark">
+                        Plan your degree from your first day on campus to your
+                        gradutation date all in one place. Ensure from the
+                        beginning that you cover every credit and prerequisite
+                        to make your college experience as smooth as possible.
+                    </div>
+                </div>
+
+                <Lottie
+                    animationData={capLottie}
+                    loop={true} // Set looping
+                    autoplay={true} // Set autoplay
+                    className="w-[350px] ml-[200px]"
+                />
+            </div>
```

Comment:
Make each section inside their own div so you can create a card for each one of them. I think this would help make it look better. Then add like a shadowy effect to each card them. I did something like this in my [personal portfolio](https://www.romericodavidjr.site/). 

Something like this:
<img width="302" alt="Screenshot 2024-12-01 at 9 53 16 PM" src="https://github.com/user-attachments/assets/b2e02cf7-6c18-48e3-a465-1f250eea2cb7">

ChatGPT will help you a lot with this styling (and also don't use wildly different colors from the background).

Reply:
Just to clarify, you want like the "Plan you Degree", the rest of the text, and the Lottie image all in their own cards with shadow effects? Just want to make sure I'm making the right changes before I commit to anything.

Reply:
No, don't create a separate card for each field within a section. For each section, create one large card.